schema.sql

```diff
@@ -12,3 +12,15 @@ CREATE TABLE animals (
 
 ALTER TABLE animals 
 ADD species VARCHAR(100);
+
```

Comment:
- [ ] For this project you are required to add a new table with the name `owners`. 
- [ ] For this project you are required to add a new table with the name `species`